Import/ReportingServicesConverter.cs

```diff
@@ -546,35 +546,47 @@ public class ReportingServicesConverter : ExternalConverterBase, IReportingServi
-                                        UpdateControlsReportDataSource(label, expressionParserResult);
-                                    } else
-                                        label.Text = textRunElement.Value;
-                                    break;
-                                case "Style":
-                                    ProcessStyle(textRunElement, label);
-                                    break;
-                                case "Label":
-                                    if(expressionParserResult == null)
-                                        label.NullValueText = e.Value;
-                                    break;
-                                case "MarkupType":
-                                case "ActionInfo":
-                                    break;
-                                default:
-                                    TraceInfo(Messages.RichTextRunElement_NotSupported_Format, textRunElementName);
-                                    break;
-                            }
-                        });
+                        var textRuns = e.Elements(xmlns + "TextRun").ToList();
+                        var values = new List<string>();
+
+                        foreach(var run in textRuns) {
+                            IterateElements(run, (textRunElement, textRunElementName) => {
+                                switch(textRunElementName) {
+                                    case "Value":
+                                        values.Add(textRunElement.Value);
+                                        break;
+                                    case "Style":
+                                        ProcessStyle(textRunElement, label);
+                                        break;
+                                    case "Label":
+                                        if(textRuns.Count == 0)
+                                            label.NullValueText = e.Value;
+                                        break;
+                                    case "MarkupType":
+                                    case "ActionInfo":
+                                        break;
+                                    default:
+                                        TraceInfo(Messages.RichTextRunElement_NotSupported_Format, textRunElementName);
+                                        break;
+                                }
+                            });
+                        }
+
+                        if(values.Count == 1) {
+                            ProcessLabelValue(values[0], label);
+                            break;
+                        }
+
+                        var sb = new StringBuilder();
+                        sb.Append("=");
+                        for (var i = 0; i < values.Count; i++) {
+                            var value = values[i];
+                            sb.Append(IsExpression(value) ? value.Substring(1) : $"\"{value}\"");

```

Comment:
value should be escaped: double quotes to quadruple quotes :) " -> ""

Reply:
That would not compile. The correct way to escape quotes in interpolated strings is the \ character.

```
var value = "test";
Console.Write($"\"{value}\"");
```

Writes "test" to the console

Reply:
$"\"{value.Replace("\"", "\"\"")}\""

Reply:
Aha! Sorry, I misunderstood your request. Fixed!